src/package.xml

```diff
@@ -1919,6 +1921,7 @@
         <members>SmartyStreets_Help_Text</members>
         <members>UTIL_InputFormFormFieldAltLabelDate</members>
         <members>UTIL_InputFormFormFieldAltLabelLookup</members>
+        <members>UpdatePaymentInformation</members>
```

Comment:
Is this an RD2 specific label or a common label? Either way, wondering if this needs a prefix such as RD2_, ge or common.

Reply:
add RD2 prefix to it.